src/CMakeLists.txt

```diff
@@ -120,8 +120,8 @@ endif(BUILD_GENIVI)
 aktualizr_source_file_checks(main.cc ${SOURCES} ${HEADERS})
 
 ################## INSTALL RULES
-#configure_file(distribution/sota.service.cmake distribution/sota.service)
-#install(FILES distribution/sota.service DESTINATION /etc/systemd/system  COMPONENT init)
```

Comment:
Can we get rid of `distribution/sota.service`? We don't appear to use it anymore.

Reply:
Sorry, I didn't notice that it was a real file. I'll delete it, sure.